css/style.css

```diff
@@ -70,6 +70,7 @@ div a[rel="noopener"] img {
     width: 200px;
     height: auto;
     background: #222;
+    z-index: 5;
```

Comment:
This is a very civilized z-index

Reply:
When only one element needs to above all else then I use 999999, when I need to style two elements I turn to reasonable numbers.